modules/n1ql_fts/pages/index.adoc

```diff
@@ -0,0 +1,589 @@
+                              { "field":"overview", "match": "Star War", "fuzziness":1}
+                         ]
+                       }
+                  ]
+       }
+    }}
+) 
+and m._class = "com.cb.fts.sample.entities.Movie" 
+order by score desc
+limit 20
+----
+
+And here is the final result:
+
+[source, Javascript]
+----
+[
+    {
+        "original_title": "Star Wars: The Clone Wars",
+        "score": 0.49917292328349827
+    },
+    {
+        "original_title": "Robot Chicken: Star Wars",
+        "score": 0.470914498415767
+    },
+    {
+        "original_title": "Star Wars",
+        "score": 0.46285917938243104
+    },
+    {
+        "original_title": "Plastic Galaxy: The Story of Star Wars Toys",
+        "score": 0.3743978968736112
+    },
+    {
+        "original_title": "Empire of Dreams: The Story of the Star Wars Trilogy",
+        "score": 0.36852905223275517
+    },
+    {
+        "original_title": "Rogue One: A Star Wars Story",
+        "score": 0.2805422105780084
+    },
+    {
+        "original_title": "Star Wars: The Force Awakens",
+        "score": 0.2654434073853831
+    },
+    {
+        "original_title": "The Star Wars Holiday Special",
+        "score": 0.22342599478315542
+    },
+    {
+        "original_title": "Star Wars: Episode I - The Phantom Menace",
+        "score": 0.21107972432534292
+    },
+    {
+        "original_title": "Star Wars: Episode III - Revenge of the Sith",
+        "score": 0.1972967995652525
+    },
+    {
+        "original_title": "Star Wars: Episode II - Attack of the Clones",
+        "score": 0.1972967995652525
+    },
+    {
+        "original_title": "The Star of Bethlehem",
+        "score": 0.18303281446562805
+    },
+    {
+        "original_title": "War",
+        "score": 0.10631280160006194
+    },
+    {
+        "original_title": "Star!",
+        "score": 0.09530237908969429
+    },
+    {
+        "original_title": "The Star",
+        "score": 0.09530237745866353
+    },
+    {
+        "original_title": "The War",
+        "score": 0.08606274267952997
+    },
+    {
+        "original_title": "The War",
+        "score": 0.0843752379211078
+    },
+    {
+        "original_title": "A Star Is Born",
+        "score": 0.07677374071093948
+    },
+    {
+        "original_title": "A Star for Christmas",
+        "score": 0.07677374071093948
+    },
+    {
+        "original_title": "A Star Is Born",
+        "score": 0.0738209045297495
+    }
+    ]
+----
+
+
+In the case above we also added the *"size":40* to our query. This will limit the number of results returned from the Full-Text Search engine to N1QL. 
+
+Without the *size* attribute, FTS would send +600 results to N1QL, then all those results would be evaluated by our *rankMovie* function and just the top 20 will be returned. As I expect that a movie in the 50th position even after our rankMovie function won't jump to the first positions, we could specify to FTS to return just the top 40 movies.
```

Comment:
`won't jump to the first positions` should be `won't jump to the first position`

Reply:
done